tests/rhui3_tests_lib/rhuimanagercli.py

```diff
@@ -0,0 +1,110 @@
+""" RHUIManagerCLI functions """
+
+import time
+
+from stitches.expect import Expect
+from rhui3_tests_lib.util import Util
+
+class RHUIManagerCLI(object):
+    '''
+    The RHUI manager command-line interface (shell commands to control the RHUA).
+    '''
+    @staticmethod
+    def cert_upload(connection, certificate_file, test_string):
+        '''
+        upload a new or updated Red Hat content certificate
+        '''
+        Expect.ping_pong(connection, "rhui-manager cert upload --cert " + certificate_file, test_string)
+
+    @staticmethod
+    def cert_info(connection):
+        '''
+        check the validity of the certificate
+        '''
+        Expect.ping_pong(connection, "rhui-manager cert info", "Valid")
+
+    @staticmethod
+    def repo_unused(connection, repo):
+        '''
+        check if a repo specified by its product name is available
+        '''
+        Expect.ping_pong(connection, "rhui-manager repo unused", Util.esc_parentheses(repo))
+
+    @staticmethod
+    def cert_expiration(connection):
+        '''
+        check if the certificate expiration date is OK
+        '''
+        Expect.ping_pong(connection, "rhui-manager status", "Entitlement CA certificate expiration date.*OK")
```

Comment:
@RadekBiba similar to my comment on cert_info function. I think such function should be more universal and can be used to check all cases.

Reply:
Similar response. Not much we can do at the moment.